### PR TITLE
prepare migration to 8.0

### PR DESCRIPTION
**Pull request by gurneyalex**
continue the work started in #4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+language: python
+
+python:
+    - "2.7"
+
+env:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
```
Comment (pedrobaeza) [inline .travis.yml]: Change to 8.0

**Comment by pedrobaeza** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+
+services:
+  - postgresql
```
Comment (bwrsandman) [inline .travis.yml]: This section is not necessary

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[![Build Status](https://travis-ci.org/OCA/product-attribute.svg?branch=master)](https://travis-ci.org/OCA/product-attribute)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/product-attribute.svg)](https://coveralls.io/r/OCA/product-attribute?branch=master)
```
Comment (bwrsandman) [inline README.md]: Please use the png badge with the branch specifier

**Comment by bwrsandman** [comment]
Since this PR is preparing for 8.0, why not put the badges towards the 8.0 branch instead of master

**Comment by bwrsandman** [comment]
:+1:

**Comment by lepistone** [comment]
:+1:

**Pull request merged**